logagg/forward/forwarder.py

```diff
@@ -3,22 +3,22 @@
 import Queue
 from threading import Thread
 
-import nsq
 import pymongo
 from pymongo import MongoClient
+from nsq.reader import Reader
 from basescript import BaseScript
 
 class LogForwarder(BaseScript):
     DESC = "Gets all the logs from nsq and stores in the storage engines"
 
     MAX_IN_FLIGHT = 100 # Number of messages to read from NSQ per shot
-    QUEUE_MAX_SIZE = 2000
+    QUEUE_MAX_SIZE = 5000
     SERVER_SELECTION_TIMEOUT = 500 # MongoDB server selection timeout
 
     SLEEP_TIME = 1
     QUEUE_TIMEOUT = 1
-    MAX_SECONDS_TO_PUSH = 2
-    MAX_MESSAGES_TO_PUSH = 100
+    MAX_SECONDS_TO_PUSH = 5
```

Comment:
@praneeth36 it doesn't make sense to have 5 seconds.
We should have it as one second, i.e we should write messages whenever we met the condition `min(reached 100 messages, completed one second)`.